### ISSUE TITLE
ISSUE: Event Data missing

### ISSUE THREAD

**Issue by tybo611** [raman325/lock_code_manager#43]
**Describe the bug**
Event data doesn't populate the code slot names correctly.  Shows as blank, hoping I did something wrong but I'm missing it right now. 
**Environment (please complete the following information):**

-   OS: HassOS
-   Type of system that HA is running on: VirtualBox
-   Home Assistant version: 2024.3
-   Hassio/Docker/Core? Core
-   Component version: [e.g. 0.1.2]
-   Lock integration name: [e.g. zwave_js] zwave_js
-   Lock make and model: [e.g. Schlage BE469] schalge BE469

**Logs**
Paste your error logs here.

**Screenshots**
![Screenshot 2024-03-01 165645](https://github.com/raman325/lock_code_manager/assets/77474705/5dc68e73-af07-4af6-be48-c9bfbe01f101)
![Screenshot 2024-03-01 165621](https://github.com/raman325/lock_code_manager/assets/77474705/ce269008-b964-43c5-a18d-80025b322363)

**Additional context**
Please add emails in plain/text format if possible and applicable.

**Comment by raman325** [comment]
thanks for the report, are you able to test the linked PR to see if it resolves the issue? It does on my end but another confirmation would be nice. I am missing tests for the event but will add them later

**Comment by tybo611** [comment]
Should be able to, I'll try to get it tested today.  

Thank you for the work

**Comment by tybo611** [comment]
> thanks for the report, are you able to test the linked PR to see if it resolves the issue? It does on my end but another confirmation would be nice. I am missing tests for the event but will add them later

Tested with both locks I have and multiple codes, both now report the name when event is fired and can be used in automation for notification.

**Comment by tybo611** [comment]
@raman325 Not 100%, but i think the code for this fix was overwritten with the fix for hard refresh.  Still looking through but lost the ability to determine which user code was used.

**Comment by raman325** [comment]
why do you think it was that PR? It didn't touch the event code, and my test for slot name is still passing with what's in `main`

**Comment by tybo611** [comment]
It's my setup, not the PR. I originally manually moved the two modified files(zwave and base) for testing, after I saw a few other bug fixes I re-downloaded the latest main branch.  Doing so I lost all modifications even though I downloaded the 0.2.5.  Looking at the main branch, none of what was online was included in the recent download.  I just manually moved the files again and everything is working.
